components/forms/conditionOfJoining/CojView.tsx

```diff
@@ -0,0 +1,64 @@
+import { Formik } from "formik";
+import { Button } from "nhsuk-react-components";
+import { useEffect } from "react";
+import { signCoj } from "../../../redux/slices/traineeProfileSlice";
+import { updatedsigningCojProgName } from "../../../redux/slices/userSlice";
+import store from "../../../redux/store/store";
+import history from "../../navigation/history";
+import ScrollTo from "../ScrollTo";
+import CojGg9 from "./CojGg9";
+
+const CojView: React.FC = () => {
+  const progName = store.getState().user.signingCojProgName;
```

Comment:
This is more of TODO item to look at a for future ticket than a request for changes. 
We need to look to see if we we are making best use of the `signingCojProgName` state variable to track the user's progress in signing the form. The original  idea was to prevent users from bypassing the CojView btn onSubmit. And the `return` statement in the `useEffect` resetting the `signingCojProgName` to null when leaving the page.